src/SFA.DAS.ApplyService.Data/Queries/OversightReviewQueries.cs

```diff
@@ -45,7 +45,9 @@ public async Task<PendingOversightReviews> GetPendingOversightReviews(string sea
                               FROM Apply apply
 	                      INNER JOIN Organisations org ON org.Id = apply.OrganisationId
                          LEFT JOIN OversightReview r ON r.ApplicationId = apply.ApplicationId
+                         LEFT JOIN Appeal Appeal on apply.ApplicationId = Appeal.ApplicationId
 	                      WHERE apply.DeletedAt IS NULL
+                          AND Appeal.ApplicationId IS NULL
```

Comment:
The oversight review check below is filtering on Status, can you match it to do the same check: `Appeal.Status IS NULL` or do `r.ApplicationId IS NULL`

Reply:
changed to Appeal.Status